helm/cluster-apps-operator/values.yaml

```diff
@@ -1,6 +1,6 @@
 appOperator:
   catalog: control-plane-catalog
-  version: 6.6.4
+  version: 6.8.1
```

Comment:
Would it make sense to pass this value using the `config` repo instead? that way we don't need a new release every time we need to upgrade.

Reply:
Hmm, I think it should be doable, let me try